packages/databricks-vscode/DATABRICKS.quickstart.md

```diff
@@ -10,95 +6,117 @@ The Databricks extension for VS Code allows you to develop for the Databricks La
+    -   [What is databricks.yml?](#what-is-databricksyaml)
+    -   [No environment variables in terminals](#no-env-vars)
 
 ---
 
 # <a id="setup-steps"></a>Getting Started
 
-## <a id="configure-extension"></a>Configure Extension
+## <a id="create-databricks-project"></a>Create a Databricks Project
 
 1. Open the Databricks panel by clicking on the Databricks icon on the left
-2. Click the "Configure Databricks" button
-3. Follow the wizard to select or configure a CLI profile
-4. Click the "gear" icon in the clusters tree item to select an interactive cluster for running code on
-    1. You can also select the first entry in the list to create a new cluster. Selecting the item will take you into the Databricks web application.
-    2. We recommend creating a Personal Compute Cluster.
-5. Click the "gear" icon in the Repo tree item to select a repo to sync code to
-    1. You can also select the first entry in the list to create a new Databricks repo
+2. Click on the "Create a new Databricks Project" button.
+3. Follow the wizard to select an existing Databricks Profile or create a new one.
+4. Select a folder to create your project in.
+5. Follow the wizard to create a new Databricks Project.
+6. Select the newly created project to open it, using the selector that appears.
+7. VS Code will reopen with the new project loaded, and the extension will automatically login using the selected profile.
+
+![create-databricks-project](./images/dabs_vsc.gif)
+
+If your folder has multiple [Databricks Asset Bundles](#dabs), you can select which one to use by clicking "Open Existing Databricks Project" button and selecting the desired project.
+
+## <a id="select-cluster"></a>Select a cluster
 
-![configure](./images/configure.gif)
+The extension uses an interactive cluster to run code. To select an interactive cluster
+
+1. Open the Databricks panel by clicking on the Databricks icon on the left
+2. Click on the "Select Cluster" button.
+    - If you wish to change the selected cluster, click on the "Configure Cluster" gear icon, next to the name of the selected cluster.
 
 ## <a id="running-code"></a>Running Code
 
-Once you have your project configured you can sync your local code to the repo and run it on a cluster. You can use the https://github.com/databricks/ide-best-practices repository as an example.
+Once you have your project configured you can deploy your local code to the selected Databricks workspace and run it on a cluster.
 
 ### <a id="running-pyspark-code"></a>Running PySpark code
 
 1. Create python file
 2. Add PySpark code to the python file.
-3. Click the "Run" icon in the tab bar and select "Upload and Run File on Databricks"
-
-This will start the code synchronization and run the active python file on the configured cluster. The result is printed in the "debug" output panel.
+3. Click the "Databricks Run" icon in the tab bar and select "Upload and Run File on Databricks"
 
-![run](./images/run.gif)
```

Comment:
Let's add a screen recording showcasing the flow?

Reply:
Would be great it we could add some more visuals to it (if possible)